src/components/TransactionDetail.tsx

```diff
@@ -0,0 +1,231 @@
+                  <Typography fontWeight={600}>{transactionData.subject}</Typography>
+                </Stack>
+                <Stack>
+                  <Typography sx={{ color: theme.palette.text.secondary }}>
+                    {t('app.transactionDetail.debtor')}
+                  </Typography>
+                  <Typography fontWeight={600}>{transactionData.owedBy}</Typography>
+                  <Typography fontWeight={600}>({transactionData.owedByFiscalCode})</Typography>
+                </Stack>
+              </Stack>
+            </Grid>
+            <Grid item xs={6}>
+              <Stack pt={4}>
+                <Typography sx={{ color: theme.palette.text.secondary }}>
+                  {t('app.transactionDetail.creditorEntity')}
+                </Typography>
+                <Typography fontWeight={600}>{transactionData.creditorEntity}</Typography>
+                <Typography fontWeight={600}>{transactionData.creditorFiscalCode}</Typography>
+              </Stack>
+              <Stack pt={2}>
+                <Typography sx={{ color: theme.palette.text.secondary }}>
+                  {t('app.transactionDetail.noticeCode')}
+                </Typography>
+                <Typography fontWeight={600}>{transactionData.noticeCode}</Typography>
+              </Stack>
+            </Grid>
+            <Grid item xs={12} pt={3}>
+              <Divider orientation="horizontal" />
+            </Grid>
+            <Grid item xs={12}>
+              <Stack pt={4} pb={4} alignItems={'end'} spacing={2}>
+                <Stack alignItems={'end'}>
+                  <Typography sx={{ color: theme.palette.text.secondary }}>
+                    {t('app.transactionDetail.partialAmount')}
+                  </Typography>
+                  <Typography fontWeight={600}>
+                    {transactionData.partialAmount.toFixed(2)} €
+                  </Typography>
+                </Stack>
+                <Stack alignItems={'end'}>
+                  <Typography sx={{ color: theme.palette.text.secondary }}>
+                    {t('app.transactionDetail.commission') +
+                      ' (' +
+                      t('app.transactionDetail.appliedBy') +
+                      ' ' +
+                      transactionData.PSP +
+                      ')'}
+                  </Typography>
+                  <Typography fontWeight={600}>{transactionData.fee.toFixed(2)} €</Typography>
+                </Stack>
+                <Stack alignItems={'end'}>
+                  <Typography sx={{ color: theme.palette.text.secondary }}>
+                    {t('app.transactionDetail.total')}
+                  </Typography>
+                  <Typography variant="h6">
+                    <b>{transactionData.total.toFixed(2)} €</b>
+                  </Typography>
+                </Stack>
+              </Stack>
+            </Grid>
+          </Grid>
+        </Box>
+        <Stack>
+          <Typography fontSize={16} fontWeight={600}>
```

Comment:
For now it's OK, cause I suppose there is a bug in Mui-italia... in this place I should expect a use of
`variant="body2" `